Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN . /.versions && \
 ## Install plugins (already as the specified user)
 RUN sudo -iu kubectl bash -c 'set -e; \
       helm plugin install https://github.com/databus23/helm-diff; \
-      helm plugin install https://github.com/jkroepke/helm-secrets --version v3.12.0; \
+      helm plugin install https://github.com/jkroepke/helm-secrets; \
```

Comment:
WDYT on pinning plugin versions and configuring them via `.versions`?

Reply:
Version pinning is the correct option to get reproducible builds for sure, but I think we may get back to it when needed. I'm unaware of using helm from this image in our projects.